packages/elements/src/utils/isElementBelowLimit.ts

```diff
@@ -33,10 +35,6 @@ function isElementBelowLimit(element: Element, verifyLink = true) {
   const limit = FULLBLEED_HEIGHT * 0.8 - DANGER_ZONE_HEIGHT;
   const { x, y, width, height, rotationAngle } = element;
   const points = getCorners(rotationAngle, x, y, width, height);
-  return (Object.keys(points) as Array<Extract<keyof Corners, Corners>>).some(
-    (point) => {
-      return (points[point] as typeof points[keyof typeof points]).y > limit;
-    }
-  );
+  return Boolean(Object.values(points).find((coord) => coord.y > limit));
```

Comment:
Thanks for simplifying this one 😄 

But why `Boolean( .find() )` when `.some()` exists?